include/aws/http/private/websocket_impl.h

```diff
@@ -72,6 +76,18 @@ struct aws_websocket_handler_options {
     bool is_server;
 };
 
+struct aws_websocket_client_bootstrap_function_table {
```

Comment:
They're not called vtables because that's for overriding the functions of the object itself. These are for overriding the functions that this object uses to call into external systems.

Not saying "function_table" is the greatest name. etables? xtables? evtable?

Reply:
Hmmm, yeah, that's a fair point. I rescind judgement!